src/workspaces/WorkspacePolicies/WorkspacePolicies.tsx

```diff
@@ -0,0 +1,89 @@
+import { icon, useUniqueId } from '@terra-ui-packages/components';
+import _ from 'lodash/fp';
+import pluralize from 'pluralize';
+import React from 'react';
+import { ReactNode, useState } from 'react';
+import { BillingProject } from 'src/billing-core/models';
+import { Checkbox, LabeledCheckbox } from 'src/components/common';
+import colors from 'src/libs/colors';
+import * as Style from 'src/libs/style';
+import { getPolicyDescriptions, phiTrackingLabel, WorkspaceWrapper } from 'src/workspaces/utils';
+
+export type WorkspacePoliciesProps = {
+  workspace?: WorkspaceWrapper;
+  billingProject?: BillingProject;
+  title?: string;
+  policiesLabel?: string;
+  policiesLink?: ReactNode;
+  endingNotice?: ReactNode;
+  noCheckboxes?: boolean;
+  togglePhiTracking?: (selected: boolean) => void;
+  togglePhiTrackingChecked?: boolean;
+};
+
+export const WorkspacePolicies = (props: WorkspacePoliciesProps): ReactNode => {
+  const policyDescriptions = getPolicyDescriptions(props.workspace, props.billingProject);
+  const [phiTracking, setPhiTracking] = useState(!!props.togglePhiTrackingChecked);
```

Comment:
To make it easier to remember to always call the wrapped version, could do something like this?

```suggestion
  const [phiTracking, _setPhiTracking] = useState(!!props.togglePhiTrackingChecked);
  const setPhiTracking = useCallback((selected) => {
    _setPhiTracking(selected);
    props.togglePhiTracking?.(selected);
  }, []);
```

Reply:
I thought a leading underscore means something is unused. I'm having trouble understanding why we want to use it here.

Reply:
Note that I do have a unttest to make sure that the callback is triggered.

Reply:
Yeah. Maybe a custom hook would be useful for this pattern since it appears in a few places.

I'll still claim the `on...` naming convention makes it easier to think about the flow of interactions / state, but I won't push it 🤷.

Reply:
I did change it to `onTogglePhiTracking`.